convert/src/vertex_to_bd/badgerdoc_format/annotation_converter.py

```diff
@@ -0,0 +1,108 @@
+from typing import Any, List, Optional, Tuple
+
+from ..models import LabelStudioModel, ModelItem, ResultItem
+from ..models.bd_annotation_model import BadgerdocAnnotation, Obj, Page, Size
+from ..models.bd_tokens_model import Page as BadgerdocTokensPage
+
+
+class AnnotationConverter:
+    def __init__(self):
+        pass
+
+    def convert(
+        self,
+        annotations: LabelStudioModel,
+        badgerdoc_tokens: BadgerdocTokensPage,
+    ):
+        model_items: List[ModelItem] = annotations.__root__
+        badgerdoc_annotations = BadgerdocAnnotation(
+            pages=[Page(page_num=1, size=Size(width=100, height=100))]
+        )
+        for model_item in model_items:
+            for annotation in model_item.annotations:
+                for item in annotation.result:
+                    self.process_labels(
+                        badgerdoc_annotations, item, badgerdoc_tokens
+                    )
+                for item in annotation.result:
+                    self.process_relations(badgerdoc_annotations, item)
+        return badgerdoc_annotations
+
+    def process_labels(
+        self,
+        badgerdoc_annotations: BadgerdocAnnotation,
+        result: ResultItem,
+        badgerdoc_tokens: BadgerdocTokensPage,
+    ):
+        if result.type != "labels":
+            return
+
+        value = result.value
+        id = len(badgerdoc_annotations.pages[0].objs)
+        for label in value.labels:
+            id += 1
+            type = "text"
+            # TODO: check start, end
```

Comment:
> Either way, I believe "TODO" probably shouldn't be in production code.
I agree with that statement, but this PR is actually not finished yet, because we were in a hurry. Annotation part will be improved. And I offer to save this TODOs

Reply:
My suggestion is this then:
let's make TODO's more descriptive so it could be understood by future possible team members

Reply:
https://github.com/epam/badgerdoc/issues/169